src/core_modules/capture-core/components/DataEntry/dataEntryField/internal/dataEntryField.utils.ts

```diff
@@ -24,7 +23,7 @@ export function getValidationError(value: any, validatorContainers: ?Array<Valid
             return false;
         }
 
-        errorMessage = (result && result.errorMessage) || validatorContainer.errorMessage;
+        errorMessage = (result && (result as any).errorMessage) || validatorContainer.errorMessage;
```

Comment:
Remove `as any` and fix the error by using `in`. Something like this: 
`errorMessage = (result && 'errorMessage' in result && result.errorMessage) || validatorContainer.errorMessage;`